package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "scripts": {
+    "test": "sass-lint static/**/*.scss --verbose --no-exit",
+    "build": "node-sass --include-path node_modules static/sass --output static/css",
+    "watch": "node-sass --include-path node_modules --source-map true --watch static/sass --output static/css",
+    "clean": "rm -rf node_modules yarn-error.log static/css static/components cache.sqlite"
+  },
+  "author": "Canonical webteam",
+  "license": "LGPL v3",
```

Comment:
yarn complains about it: `warning package.json: License should be a valid SPDX license expression`

Reply:
Since this isn't actually an NPM package, maybe I'll just remove it.